install/share/profiles/KDCs_PKINIT_Certs.cfg

```diff
@@ -0,0 +1,109 @@
+profileId=KDCs_PKINIT_Certs
```

Comment:
I wonder, why do we need a special Dogtag profile for the pkinit certs? I couldn't find an explanation on https://www.freeipa.org/page/V4/External_Authentication